build/linux/installer/conf/telegraf.conf

```diff
@@ -675,7 +675,9 @@
   ## An array of urls to scrape metrics from.
   urls = ["$CADVISOR_METRICS_URL"]
 
-  fieldpass = ["kubelet_running_pod_count","volume_manager_total_volumes", "kubelet_node_config_error", "process_resident_memory_bytes", "process_cpu_seconds_total"]
+  # <= 1.18: metric name is kubelet_running_pod_count
+  # >= 1.19: metric name changed to kubelet_running_pods
+  fieldpass = ["kubelet_running_pod_count","kubelet_running_pods","volume_manager_total_volumes", "kubelet_node_config_error", "process_resident_memory_bytes", "process_cpu_seconds_total"]
```

Comment:
Isn't the new ones 	'running_pods' & 'running_containers' ?

Reply:
Could you point me to where you see new names as 'running_pods' & 'running_containers? I have tried with 'kubelet_running_pods' and see the data in LA

Reply:
here - https://github.com/kubernetes/kubernetes/blob/a7fe0bf8e593decabba030f771ea8dbb34d22710/pkg/kubelet/metrics/metrics.go#L56

Metric is named without 'kubelet' but looks its prefixing with subsystem (kubelet in this case). If you see the metric flowing, we are good.